graph/api/src/main/java/org/jboss/windup/graph/model/ApplicationArchiveModel.java

```diff
@@ -2,7 +2,8 @@
 
 import com.tinkerpop.frames.modules.typedgraph.TypeValue;
 
-@TypeValue("ApplicationArchive")
+@TypeValue(ApplicationArchiveModel.TYPE)
 public interface ApplicationArchiveModel extends ApplicationModel, ArchiveModel
 {
+    String TYPE = "ApplicationArchiveModel";
```

Comment:
I would generally remove the Model suffix, it's not really needed and makes everything longer - esp. the JSON data, URLs etc.

Reply:
+1